package.json

```diff
@@ -49,6 +55,7 @@
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-page-title": "^6.2.2",
     "ember-qunit": "^5.1.4",
+    "ember-resize-observer-modifier": "^1.1.1",
```

Comment:
should be a dependency as you use it in your components

Reply:
https://github.com/concordnow/ember-sidenotes/pull/1/commits/cb25c92750f0c3fbac316a4d137037f7c37abdb2 :+1: